mix.exs

```diff
@@ -18,9 +18,9 @@ defmodule Calendar.Mixfile do
 
   def deps do
     [
-      {:tzdata, "~> 0.5.8 or ~> 0.1.201603"},
+      {:tzdata, "~> 0.5.11"},
```

Comment:
Tests all pass on this, but sure exactly why `or ~> 0.1.201603` was there though. Just wanted to make it easier after suggesting this bump [here](https://github.com/lau/tzdata/issues/14)